deployments/ansible/roles/signalfx-agent/tasks/yum_repo.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Add Signalfx Agent repo into source list
+  yum_repository:
+    name: signalfx-agent
+    description: SignalFx Agent Repository
+    baseurl: "{{ rhel_repo_url }}"
+    gpgkey: "{{ rhel_gpg_key_url }}"
+    gpgcheck: yes
+    enabled: yes
+
+- name: Install Signalfx Agent via yum package manager
```

Comment:
All of the logic here and below is repeated from the debian task file.  Pull the common steps out to the main.yml file so that there is no repetition.

Reply:
sure, moved common tasks to main